scripts/sample_export_config.json

```diff
@@ -113,6 +113,26 @@
       "name": "sodium_100g",
       "nutrient_id": 1093,
       "scale": 0.001
-    }
+    },
+    {
+      "name": "calcium_100g",
+      "nutrient_id": 1087,
+      "scale": 0.001
+    },
+    {
+      "name": "iron_100g",
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_IU_100g",
+      "nutrient_id": 1089,
+      "scale": 1
+    },
+    {
+      "name": "vitamin-C_IU_100g",
```

Comment:
I believe you want lowercase here (`vitamin-c`) based on `https://static.openfoodfacts.org/data/data-fields.txt`.  Or if you have some other convention for imports, `vitamin-a` above is lowercase.